cli/create.go

```diff
@@ -129,6 +115,23 @@ func create() *cobra.Command {
 				}
 			}
 
+			schedSpec, err := validSchedule(
+				autostartMinute,
+				autostartHour,
+				autostartDow,
+				tzName,
+				time.Duration(template.MinAutostartIntervalMillis)*time.Millisecond,
+			)
+			if err != nil {
+				return xerrors.Errorf("Invalid autostart schedule: %w", err)
+			}
+			if ttl < time.Minute {
+				return xerrors.Errorf("TTL must be at least 1 minute")
+			}
+			if ttlMax := time.Duration(template.MaxTTLMillis) * time.Millisecond; ttl > ttlMax {
+				return xerrors.Errorf("TTL must be below template maximum %s", ttlMax)
+			}
+
```

Comment:
Review: We're duplicating some validations here because I want the errors to show up _before_ the whole workspace plan happens.

Reply:
Interesting to do it in the cli. Can we somehow put the validation on a datastructure and reuse it for the backend + frontend? Like reuse the same code?

Not a huge deal.

Reply:
It gets validated on the BE as well but if we don't at least validate in the `create` command the user ends up seeing the workspace dry run, confirming the create, and _then_ getting the validation error, which is annoying.

And yes, this is definitely a good candidate to be DRYed up a bit.